README.md

```diff
@@ -212,6 +212,15 @@ Official documenation is [here](https://confluence.jetbrains.com/display/TCD18/C
 
 From now on, new commits and pull requests will have TeamCity status shown!
 
+## Restarting disconnected agent
+
+1. Remove the disconnected agent - via teamcity site go to disconnected list of agents and remove the agent
+1. On the command line ssh to montagu as vagrant user `ssh vagrant@support.montagu.dide.ic.ac.uk` and destroy the removed agent `vagrant destroy <agent_name>`
+1. Delete the disk for the agent `vboxmanage closemedium disk disk/<agent_name>.vdi --delete
```

Comment:
I'm not sure this is necessary - I normally just `vagrant reload <agent_name>`

Reply:
It depends on why the agent has died - I think sometimes we've had problems where the disk has filled up, and this clears that out

Reply:
I wonder if we should modify the provision script to wipe the disk - that would mean that Alex's version would work, which would be simpler. Just before this line https://github.com/vimc/montagu-ci/blob/master/provision/setup-docker.sh#L11 add a 

```
rm -rf /mnt/data/docker/var-lib-docker
```

and that should be enough, then replace the destroy/delete lines with Alex's change

Reply:
Right I'm a bit confused about what this should look like then. I can make the change to the `setup-docker.sh` script. 

Does this mean the instructions should be
1. Remove disconnected agent
2. `vagrant reload` the agent
3. Authorise the agent again?

Reply:
Based on our chat last night, I think we do need destroy/up as you have here, but I think we can replace the disk faffery with the change to setup-docker.sh script - add the line I posted above at line 11